webpack/localization-plugin/src/LocalizationPlugin.ts

```diff
@@ -655,4 +724,41 @@ export class LocalizationPlugin implements Webpack.Plugin {
       suffix: suffix
     };
   }
+
+  private _chunkHasLocalizedModules(chunk: Webpack.compilation.Chunk): boolean {
+    if (EntityMarker.getMark(chunk) === undefined) {
+      let chunkHasAnyLocModules: boolean = false;
+      if (!chunkHasAnyLocModules) {
+        for (const module of chunk.getModules()) {
+          if (EntityMarker.getMark(module)) {
+            chunkHasAnyLocModules = true;
+            break;
+          }
+        }
+      }
+
+      // Check async chunks if this is a runtime chunk and we haven't directly found any localized modules
```

Comment:
Surely the marker for this state should be different from the marker for the chunk itself needing localization? This state just necessitates a tweak to the runtime definition to reroute the loading.

Reply:
You're referring to the case where an entrypoint doesn't directly contain any localized resources, but it references async chunks that do?

In that case, we do modify the runtime definition to reroute the loading, but the locale name for the async chunks needs to land somewhere in the entrypoint chunk so, therefore, the entrypoint ends up also becoming localized.

Reply:
My concern is about the chunks between the entry point chunk and the localized chunks.

Reply:
Only the entrypoint chunk(s) contain(s) the async chunk loading code, so only the entrypoint will end up localized.